cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/data/PartitionedDataLayer.java

```diff
@@ -165,6 +162,10 @@ public static void validateReplicationFactor(@NotNull ConsistencyLevel consisten
         {
             return;
         }
+
+        Preconditions.checkArgument(!ConsistencyLevel.EACH_QUORUM.equals(consistencyLevel),
```

Comment:
Given it's an enum we can do:

```suggestion
        Preconditions.checkArgument(ConsistencyLevel.EACH_QUORUM != consistencyLevel,
```